man/rgbasm.5

```diff
@@ -1804,11 +1804,28 @@ The
 value will be updated by
 .Ar step
 until it reaches or exceeds
-.Ar stop .
+.Ar stop ,
+i.e. it covers the half-open range from
+.Ar start
+(inclusive) to
+.Ar stop
+(exclusive).
+The variable
+.Ar V
+will be assigned this value at the beginning of each new iteration; any changes made to it within the
+.Ic FOR
+loop's body will be overwritten.
+So if the symbol
+.Ar V
+is already defined before the
+.Ic FOR
+loop begins, it must be a variable
+.Pq Sx Variables .
```

Comment:
```suggestion
So the symbol
.Ar V
need not be already defined before any iterations of the
.Ic FOR
loop, but it must be a variable
.Pq Sx Variables
if so.
```
This clarifies that the symbol is defined at the beginning of *every* iteration, and that it must either not exist or be a variable as well. (Might be worth pointing out that the symbol stays defined afterwards, though the example can suffice for that.)

Reply:
Okay.